coderd/schedule/autostart.go

```diff
@@ -28,3 +32,19 @@ func NextAutostart(at time.Time, wsSchedule string, templateSchedule TemplateSch
 
 	return zonedTransition, allowed
 }
+
+func NextAllowedAutostart(at time.Time, wsSchedule string, templateSchedule TemplateScheduleOptions) (time.Time, error) {
+	next := at
+
+	// Our cron schedules work on a weekly basis, so to ensure we've exhausted all
```

Comment:
```suggestion
	// Our autostart day requirements may block the day, so to ensure we've exhausted all
```
Maybe this is entirely wrong, but it's what I would expect the comment to be as I'm having trouble understanding it. Otherwise what does cron schedules/weekly basis mean?

Reply:
I can see how my comment is poorly worded. 

What I was trying to say is that since the cron schedule is parsed with `cron.Weekly` it is scoped to a single-week (so you cant specify first Monday of the month for example). So the basis of the comment is saying we only need to check the next 7 days to have fully exhausted the set of possible autostarts.

Reply:
Gotcha, that makes sense, thanks for clarifying 👍🏻.